addons/website/static/src/scss/website.scss

```diff
@@ -1656,6 +1678,7 @@ header {
     .o_mega_nav {
         padding-left: $grid-gutter-width * .5;
         height: $o-mega-menu-nav-height;
+        background-color: inherit;
```

Comment:
Ideally, it would have taken the background color of the mega menu, but that's not possible due to the DOM structure.

Reply:
Mmh I am not even sure that I would prefer the mega menu color. First: hopefully the user choose a mega menu color which is related to its menu color, then the two colors should go well together and its good to have the menu controls inside the menu colors IMO.